locale/en/support-ukraine.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: support-ukraine.hbs
+---
+
+<iframe width="100%" height="400" src="https://www.youtube.com/embed/SDJHUMzIlLo" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
We can use the "privacy-enhanced mode" of YouTube embed, which (is supposed to) give folks the ability to opt-in to Google tracking rather than being forced into it when loading the page.
```suggestion
<iframe width="100%" height="400" src="https://www.youtube-nocookie.com/embed/SDJHUMzIlLo" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```